source/test/home-page.test.js

```diff
@@ -0,0 +1,72 @@
+const { count } = require("console");
+const { PRIORITY_BELOW_NORMAL } = require("constants");
+
+describe('Basic user flow for Website', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8453404)
```suggestion
describe("Basic user flow for Website", () => {
```